crates/rolldown/src/bundler.rs

```diff
@@ -51,19 +52,25 @@ impl Bundler {
 impl Bundler {
   #[tracing::instrument(level = "debug", skip_all, parent = &self.build_span)]
   pub async fn write(&mut self) -> BuildResult<BundleOutput> {
+    trace_action!(action::BuildStart { kind: "BuildStart" });
     let scan_stage_output = self.scan(vec![]).await?;
 
-    self.bundle_write(scan_stage_output).await
+    let ret = self.bundle_write(scan_stage_output).await;
+    trace_action!(action::BuildEnd { kind: "BuildEnd" });
+    ret
   }
 
   #[tracing::instrument(level = "debug", skip_all, parent = &self.build_span)]
   pub async fn generate(&mut self) -> BuildResult<BundleOutput> {
+    trace_action!(action::BuildStart { kind: "BuildStart" });
```

Comment:
I realize that these could be abstract as  the start and end of a `Span`. Would optimize them in the future.